template/App.tsx

```diff
@@ -12,6 +12,7 @@ import {useStreamChatTheme} from './useStreamChatTheme';
 import {AppStateProvider} from './AppContext';
 import {useStreamChat} from './useStreamChat';
 import {Navigation} from './Navigation';
+import { GestureHandlerRootView } from "react-native-gesture-handler";
```

Comment:
Gonna move this together with the other external imports, and remove the spaces within the brackets before merging this